docs/developer_guide/env_setup.md

```diff
@@ -109,6 +109,29 @@ Without setting the enviromnent minikube might not find the docker images when d
 1. run the image build from the yunikorn-k8shim repository root: `make image`
 1. deploy the scheduler as per the normal instructions.
 
+## Local Kubernetes Cluster with Kind
+
+Kind (Kubernetes in Docker) is a lightweight tool for running lightweight Kubernetes environments.  It is very easy to test different Kubernetes versions with kind.  You can just select the kind image you want.
+
+### Installation
+
+If you have go installed, you can run `go install sigs.k8s.io/kind@latest`.
+
+Other ways can be found on the Kind [website](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
+
+### Using Kind
+
+To test a new version of Kubernetes, you can pull a corresponding image from kind's repo.
+
+Creating a 1.23.13 Kubernetes Cluster: `kind create cluster --name test --image kindest/node:v1.23.13`
+
+Deleting a kind cluster: `kind delete cluster --name test`
+
+### Loading your images
+
+In order to use a local image, you have to load your images into kind's registry.  If you run `make image`, you could use the following command to load your kind image.  This assumes AMD64 architecture.
```

Comment:
Extend this to mention all images that could be used (admission, scheduler, plugin and web), not just the scheduler